code_samples/simple_language_plugin/testData/AnnotatorTestData.java

```diff
@@ -2,6 +2,6 @@
   public static void main(String[] args) {
     System.out.println("simple:website");
     System.out.println("simple:key with spaces");
-    System.out.println("simple:<error descr="Unresolved property">websit"</error>);
+    System.out.println("simple:<error descr=\"Unresolved property\">website</error>");
```

Comment:
The correct statement is:
`System.out.println("simple:<error descr="Unresolved property">websit</error>");`

See [Testing Highlighting](https://www.jetbrains.org/intellij/sdk/docs/basics/testing_plugins/testing_highlighting.html)

Reply:
for JAVA code you need to escape all "(quotation mark) into (quotation marks)""
you can try to run tests with existing code, they will not work